libs/test-cli/src/main.rs

```diff
@@ -414,8 +439,11 @@ async fn migrate_diff(cmd: &MigrateDiff) -> anyhow::Result<()> {
 
     let api = schema_core::schema_api(None, Some(Arc::new(DiffHost)))?;
     let to = if let Some(to_schema_datamodel) = &cmd.to_schema_datamodel {
-        DiffTarget::SchemaDatamodel(SchemaContainer {
-            schema: to_schema_datamodel.clone(),
+        DiffTarget::SchemaDatamodel(SchemasContainer {
+            files: vec![SchemaContainer {
+                path: "schema.prisma".to_string(),
+                content: to_schema_datamodel.clone(),
+            }],
         })
```

Comment:
Fix: `to_schema_datamodel` is likely a path. We need to read the file and pass it in as `content`.